### PR TITLE
Legalese

### PR DESCRIPTION
**Pull request by rakvat**
I took content from desk and manually converted it to markdown.

### CODE REVIEW

**Comment by rakvat** [comment]
@johnbaku fixed!

File: legalese/2257_exempt/en.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# 2257 Exempt
+
+## 1. JURISDICTION
+PLEASE NOTE: By virtue of their locale the producers of the adult entertainment contained in this website do not fall under the jurisdiction of American law. The producers of this website have taken it upon themselves to voluntarily comply with United States law concerning 18 United States Code 2257 and 28 Consolidated Federal Regulations Part 75 in order to assure those that are bound by these United States laws and regulations that all visual depictions displayed on this website, whether of actual sexually explicit conduct, simulated sexual conduct or otherwise, are visual depictions of persons who were at least 18 years of age when those visual depictions were created. Government issued photographic identification confirming that the foregoing is held by the Custodian of Records listed below. Such identification may or may not have been issued by the state or federal government of the USA. It is important to note that this 18 USC 2257 compliance notice is included in good faith only and for reference purposes.
```
Comment (johnbaku) [inline legalese/2257_exempt/en.md]: Please add space after header. Sorry about being picky here.

Comment (rakvat) [inline legalese/2257_exempt/en.md]: @johnbaku no problem at all. Fixed.

Comment (johnbaku) [inline legalese/2257_exempt/en.md]: 👍

**Pull request merged**